MixItUp.Base/Statistics/StatisticsTracker.cs

```diff
@@ -50,7 +50,14 @@ public StatisticsTracker()
             this.Statistics.Add(new TrackedNumberStatisticDataTracker("Viewers", "Eye", (StatisticDataTrackerBase stats) =>
             {
                 TrackedNumberStatisticDataTracker numberStats = (TrackedNumberStatisticDataTracker)stats;
-                numberStats.AddValue((int)ChannelSession.Channel.viewersCurrent);
```

Comment:
The "ChannelSession.Channel" is null when you aren't yet connected.  But this code runs, so it was null ref'ing on the background thread. I figured it'd be better to politely handle it.